src/components/Section/Psychological/Diagnoses/Diagnosis.jsx

```diff
@@ -79,20 +79,10 @@ export default class Diagnosis extends ValidationElement {
     const applicantBirthdate = getContext().applicantBirthdate
     return (
       <div className="diagnosis">
-        <Field
-          title={i18n.t(`psychological.${prefix}.heading.condition`)}
-          scrollIntoView={this.props.scrollIntoView}>
-          <Show when={this.props.prefix === 'existingConditions.diagnosis'}>
-            <Text
-              name="Condition"
-              className="diagnosis-condition"
-              {...this.props.Condition}
-              onUpdate={this.updateCondition}
-              onError={this.props.onError}
-              required={this.props.required}
-            />
-          </Show>
-          <Show when={this.props.prefix === 'diagnosis'}>
+        <Show when={this.props.prefix !== 'existingConditions.diagnosis'}>
```

Comment:
This is a pervasive pattern across the app; i.e. conditional behavior within a re-usable component based on location of component instance, but I think we will need to revisit as we start implementing different forms. It should probably be another non-path related property that gets set on instantiation.